### ISSUE TITLE
Mantle in the handbook crashes the game

### ISSUE THREAD

**Issue by Craluminum2413** [anegostudios/VintageStory-Issues#890]
**Game Version:** 1.15.0-pre1
**Platform:** Windows
**Modded:** No

### Description
The game crashes when you try to view the statistics of mantle block

### How to reproduce
1. Open handbook
2. Find the mantle and open its page
3. It crashes the game

### Expected behavior
It shows the page of mantle

### Logs
```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.15.0-pre.1 (Unstable)05/24/2021 17:00:15: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.API.Common.CollectibleObject.GetHandbookInfo(ItemSlot inSlot, ICoreClientAPI capi, ItemStack[] allStacks, ActionConsumable`1 openDetailPageFor)
   at Vintagestory.API.Client.GuiHandbookItemStackPage.GetPageText(ICoreClientAPI capi, ItemStack[] allStacks, ActionConsumable`1 openDetailPageFor)
   at Vintagestory.GameContent.GuiDialogHandbook.initDetailGui()
   at Vintagestory.API.Client.GuiElementHandbookList.OnMouseUpOnElement(ICoreClientAPI api, MouseEvent args)
   at Vintagestory.API.Client.GuiElement.OnMouseUp(ICoreClientAPI api, MouseEvent args)
   at Vintagestory.API.Client.GuiComposer.OnMouseUp(MouseEvent mouse)
   at Vintagestory.API.Client.GuiDialog.OnMouseUp(MouseEvent args)
   at Vintagestory.Client.NoObf.GuiManager.OnMouseUp(MouseEvent args)
   at Vintagestory.Client.NoObf.ClientMain.OnMouseUp(MouseEvent args)
   at Vintagestory.Client.NoObf.ClientPlatformWindows.Mouse_ButtonUp(Object sender, MouseButtonEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.Platform.Windows.WinGLNative.WindowProcedure(IntPtr handle, WindowMessage message, IntPtr wParam, IntPtr lParam)
   at OpenTK.Platform.Windows.Functions.DispatchMessage(MSG& msg)
   at OpenTK.Platform.Windows.WinGLNative.ProcessEvents()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _Sfca4roOb4ikuKbMulRniUxARF4._0ytRimz1XDNkGpyxJcZbGf9Oxbd(_YzYAgXNLrh0iXJRtlZnQ7TqOSZX , String[] )
   at _hNqfhVOk3ksmv0cZQgnBG2cCt6o._0ytRimz1XDNkGpyxJcZbGf9Oxbd(ThreadStart )

**Comment by tyronx** [comment]
thanks, fixed in pre.4

**Comment by copygirl** [comment]
@tyronx Does this also fix #985, which has a similar crash with "firepit (cold/in construction)".

**Comment by tyronx** [comment]
yes, it seems that is fixed as well
